src/dbnode/integration/large_tiles_test.go

```diff
@@ -103,9 +103,7 @@ func TestReadAggregateWrite(t *testing.T) {
 	require.True(t, flushed)
 	log.Info("verified data has been cold flushed", zap.Duration("took", time.Since(start)))
 
-	aggOpts, err := storage.NewAggregateTilesOptions(
-		dpTimeStart, dpTimeStart.Add(blockSizeT),
-		time.Hour, false)
+	aggOpts, err := storage.NewAggregateTilesOptions(dpTimeStart, dpTimeStart.Add(blockSizeT), time.Hour)
```

Comment:
Should probably add tests for counter writes (both with reset and without) to see that they get aggregated correctly too?

Reply:
I've covered these details in UT of `shard.AggregateTiles`. The integration tests of the process are so brittle and hard to debug that I would really want them to just cover the basic flow (including reading/writing ondisk data).